lib/controllers/.xervo-base-ignore

```diff
@@ -1,8 +1,11 @@
 .git/
 .modulus-base-ignore
+.xervo-base-ignore
 .modulusignore
+.xervoignore
 .meteor/local/
 modulus-auto-zip-*
```

Comment:
This file would be generated by the CLI [here](https://github.com/XervoIO/cli/blob/master/lib/controllers/project.js#L651). Wouldn't that make it ok to remove this line in this [commit](https://github.com/XervoIO/cli/pull/157/files#diff-e5001988c284bf49302c6986b4b88a3bL651)?